### DISCUSSION TITLE
Imad Dabbura - About

### DISCUSSION THREAD

**Discussion by giscus** [ImadDabbura/imaddabbura.github.io#1] [General]
# Imad Dabbura - About

https://imaddabbura.github.io/about.html

**Comment by ChrisLoukas007** [comment]
Dear Imad Dabbu, 

Hello , my name is Chris and I am an undergraduate Software Engineer at CEID, University of Patras and I am currently doing my thesis which is very relevant to this article you wrote "K-means Clustering: Algorithm, Applications, Evaluation Methods, and Drawbacks" .

More specifically the purpose of my thesis is to create a system in which the user will input an image in jpg format and the system will first recognize its main color and if he wants it will recognize some number of main colors that he will choose . After that based on the color or colors the system will provide the emotional expressions that this paint/image is trying to express with some accuracy . 

Regarding the first part, which is the recognition of the primary color, I would like your opinion of how I implement the kmean algorithm in order to work right . 

Also I would like your opinion about which metrics and evaluations must choose and run in order to be sure that I implement perfectly the Kmean algorithm . 

At this time , my goal is to become a Data Scientist ( I don't know if this the best choice but right now I have a feeling that this is what I want to follow ) , so every advise about my thesis or about Data Scienece it would realy help . 

Here is the link with the github code : https://github.com/ChrisLoukas007/ArtEmotionalPalette-AI/blob/1959b429686a5e45e3b40368e49c883f2b6bd7e9/k_mean%20primary%20color.ipynb
Best Regards , 
Chris Ntais
